www/templates/cdsp-config.html

```diff
@@ -0,0 +1,209 @@
+
+			<div class="control-group">
+				<label class="control-label" for="cdsp-setting-mode">Configuration</label>
+				<div class="controls">
+					<select id="cdsp-setting-mode" class="input-large" name="cdsp-mode" >
+						$_select[cdsp_mode]
+					</select>
+					<a aria-label="Help" class="info-toggle" data-cmd="info-cdspmode" href="#notarget"><i class="fas fa-info-circle"></i></a>
+					<span id="info-cdspmode" class="help-block-configs help-block-margin legend-info-help hide">
+						Off - don't use Camilla DSP<br/>
+						Custom - custom alsa_cdsp configuration is used, don't touch any alsa_cdsp or camilla configuration.<br/>
+						Else use the selected Camilla DSP configuration file. This one will be assigned to alsa_cdsp.
+					</span>
+				</div>
+
+
+
+				<label class="control-label" >Set device</label>
+				<div class="controls">
+					<div class="toggle">
+						<label class="toggle-radio" for="toggle-cdsp-playbackdevice2">YES</label>
+						$_select[cdsp_patch_playback_device1]
+						<label class="toggle-radio" for="toggle-cdsp-playbackdevice1">NO</label>
+						$_select[cdsp_patch_playback_device0]
+					</div>
+					<div style="display: inline-block; vertical-align: top; margin-top: 2px;">
+					  <a aria-label="Help" class="info-toggle" data-cmd="info-cdsp-playbackdevice" href="#notarget"><i class="fas fa-info-circle"></i></a>
+					</div>
+					<span id="info-cdsp-playbackdevice" class="help-block-configs help-block-margin legend-info-help hide">
+						Automatic set correctplayback device for the CamillaDSP configuration.<br/>
+						Disable if you want to used an other play back device then the default of moOde,<br/>
+						for example if you want to create a multi channel crossover.<br>
+					</span>
+
+				</div>
+			</div>
+
+		</fieldset>
+	</form>
+	<form class="form-horizontal" action="" method="post" enctype="multipart/form-data">
+		<fieldset>
+			<div>
+				<a data-toggle="modal" href="#remove-pipeline"><button class="btn btn-medium btn-primary">Remove</button></a>
+				<div class="modal-button-style">
+					<label for="pipelineconfig" id="choose-pipeline-cfg" class="btn btn-primary btn-medium">Import</label>
+					<input type="file" id="pipelineconfig" accept=".yml" name="pipelineconfig" style="display:none" onchange="$('#btn-import-pipeline').click();" >
+					<button id="btn-import-pipeline" class="btn btn-medium btn-primary btn-submit" type="submit" name="import" value="1"  style="display:none"/>
+				</div>
+				<button class="btn btn-medium btn-primary btn-submit" type="submit" name="export" value="1">Download</button>
+				<button class="btn btn-medium btn-primary btn-submit" type="submit" name="check" value="1">Check</button>
+				<a aria-label="Help" class="info-toggle" data-cmd="info-configuration-actions" href="#notarget"><i class="fas fa-info-circle"></i></a>
+				<div id="info-configuration-actions" class="help-block hide">
+					Remove: delete the selected configuation file.<br/>
```

Comment:
I renamed the buttons and changed the order a bit, but didn't make the upload new the last button.
I like to have the same order as with the coeffs files, only their the check but is absent so that will be the last button.